Jenkinsfile

```diff
@@ -205,7 +205,7 @@ pipeline{
 			 props = readProperties file:'data-hub/pipeline.properties';
 				 copyRPM 'Release','10.0-4'
 				setUpML '$WORKSPACE/xdmp/src/Mark*.rpm'
-				sh 'export JAVA_HOME=`eval echo "$JAVA_HOME_DIR"`;export GRADLE_USER_HOME=$WORKSPACE$GRADLE_DIR;export M2_HOME=$MAVEN_HOME/bin;export PATH=$GRADLE_USER_HOME:$PATH:$MAVEN_HOME/bin;cd $WORKSPACE/data-hub;rm -rf $GRADLE_USER_HOME/caches;set +e;./gradlew clean;./gradlew marklogic-data-hub:testAcceptance;sleep 10s;./gradlew ml-data-hub:test;./gradlew web:test;./gradlew marklogic-data-hub-central:test |& tee console.log;'
+				sh 'export JAVA_HOME=`eval echo "$JAVA_HOME_DIR"`;export GRADLE_USER_HOME=$WORKSPACE$GRADLE_DIR;export M2_HOME=$MAVEN_HOME/bin;export PATH=$GRADLE_USER_HOME:$PATH:$MAVEN_HOME/bin;cd $WORKSPACE/data-hub;rm -rf $GRADLE_USER_HOME/caches;set +e;./gradlew clean;./gradlew marklogic-data-hub:testAcceptance;sleep 10s;./gradlew marklogic-data-hub-central:test;./gradlew web:test;./gradlew ml-data-hub:test |& tee console.log;'
```

Comment:
I need the output of ./gradlew marklogic-data-hub-central:test to write to a console.log file, so that I can check if there are any compilation errors in the npm task which I'll be using later.

So the command should be as below: 
./gradlew marklogic-data-hub-central:test |& tee console.log; ./gradlew web:test;./gradlew ml-data-hub:test;

Reply:
Thanks - I wanted to see if the order in which the tests were run affected the authentication issues that mysteriously impacted the marklogic-data-hub-central tests. I'll change it back once I know that's not the case.